build.gradle

```diff
@@ -871,22 +876,30 @@ project('test:system') {
         systemProperty "imageVersion", System.getProperty("imageVersion") // pravega version.
         // bookkeeper version defaults to pravega version.
         systemProperty "pravegaBookkeeperVersion", System.getProperty("pravegaBookkeeperVersion", System.getProperty("imageVersion"))
-
+        systemProperty "bookkeeperImageVersion", System.getProperty("bookkeeperImageVersion", "latest") // // bookkeeper image version
```

Comment:
How is this different from pravegaBookkeeperVersion?

Reply:
pravegaBookkeeperVersion was intact with pravega image, so removed that parameter and introduced bookkeeperImageVersion which can be independent of pravega release.

Reply:
should it default to the pravega version?